docs/domains/index.html

```diff
@@ -1095,6 +1095,14 @@
     "Configuration": {
       "type": "object",
       "properties": {
+        "overrideDistributionStrategy": {
+          "description": "Determines how updated configuration overrides are distributed to already running WebLogic servers following introspection when the domainHomeSourceType is PersistentVolume or Image.  Configuration overrides are generated during introspection from secrets, the overrideConfigMap field, and WebLogic domain topology.Legal values are DYNAMIC (the default) and ON_RESTART.",
```

Comment:
Typo missing space after the period -> ". Legal values are DYNAMIC (the default) and ON_RESTART."